src/main/java/model/LottoStore.java

```diff
@@ -1,16 +1,14 @@
 package model;
 
-import java.util.List;
-
 public class LottoStore {
-    private static int LOTT_PRICE = 1000;
+    private static int LOTTO_PRICE = 1000;
```

Comment:
```suggestion
    private static final int LOTTO_PRICE = 1000;
```
변경되지 않도록 final 키워드를 붙여줘야겠네요